oecologia-australis.csl

```diff
@@ -260,10 +259,15 @@
   </macro>
   <macro name="publisher">
     <choose>
-      <if type="report" match="any">
-        <group delimiter=": ">
-          <text variable="publisher-place"/>
-          <text variable="publisher"/>
+      <if type="report" match="all">
+        <group delimiter=" " prefix=" ">
+          <choose>
+            <if match="any" variable="number-of-pages publisher">
+              <text variable="number-of-pages" prefix="p. " suffix="."/>
+              <text variable="publisher" prefix="ed. " suffix=", "/>
```

Comment:
@gudryan, it's also better to use CSL terms whenever possible, instead of hardcoded translations. E.g. instead of `<text variable="number-of-pages" prefix="p. " suffix="."/>`, use:

```xml
<group delimiter=" " suffix=".">
  <label variable="number-of-pages" form="short" plural="never"/>
  <text variable="number-of-pages"/>
</group>
```